src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -561,7 +563,7 @@ def delete_role_assignments(cmd, ids=None, assignee=None, role=None, resource_gr
                               assignments_client._config.subscription_id)
     assignments = _search_role_assignments(cmd.cli_ctx, assignments_client, definitions_client,
                                            scope, assignee, role, include_inherited,
-                                           include_groups=False)
+                                           include_groups=False, at_scope=True)
```

Comment:
I am hesitating on whether `--at-scope` should be exposed by `az role assignment delete`, as setting `--at-scope false` will include subscope assignments, causing accidental deletion.